Sources/StreamChat/Workers/ChannelUpdater.swift

```diff
@@ -18,8 +18,9 @@ class ChannelUpdater: Worker {
     /// - Parameters:
     ///   - channelQuery: The channel query used in the request
     ///   - isInRecoveryMode: Determines whether the SDK is in offline recovery mode
-    ///   - channelCreatedCallback: For some type of channels we need to obtain id from backend.
-    ///   This callback is called with the obtained `cid` before the channel payload is saved to the DB.
+    ///   - onChannelCreated: For some type of channels we need to obtain id from backend.
+    ///     This callback is called with the obtained `cid` before the channel payload is saved to the DB.
+    ///   - onBeforeSavingChannel: Hook to do some actions in the DB before saving the channel from the API response.
```

Comment:
Regarding my suggestion above, we can also make this one `onSavingChannel` as it seems that we are using past tense to recognise actions that completed (e.g. onChannelCreated)